create3_lidar/README.md

```diff
@@ -0,0 +1,71 @@
+# iRobot® Create® 3 LIDAR SLAM demo
+
+This example sets up LIDAR SLAM with a Create® 3 robot and Slamtec RPLIDAR spinning laser rangefinder.
+
+## Parts List
+
+* Raspberry Pi® 4[^1]
+* USB-C® to USB-C® cable[^2]
+* Slamtec RPLidar A1M8
+* USB Micro B to USB A cable
+* 4x M2.5 x 8 flat head machine screw
+* 4x M2.5 x 6 self-tapping screw
+* 10x M3 x 10 self-tapping screw
+
+## Setup
+
+### Hardware Setup
+
+1. Affix a LIDAR to your robot.
+   STLs are available to mount an RPLidar A1M8 as well as its USB adapter.
```

Comment:
Should we have an image to show where we mounted the laser bracket on the faceplate or describe with holes to screw into to match the kinematics of the launch file

Reply:
I've actually taken an image already; I am just not sure where to put it on the Internet.